components/Form/AddressLookup/AddressLookup.spec.tsx

```diff
@@ -1,35 +1,294 @@
-import { defaultValidation } from './AddressLookup';
+import { defaultValidation, AddressBox } from './AddressLookup';
```

Comment:
The changes you've made in this test file is such an improvement, you made broken up a big test and made it clearer to understand and you've added missing coverage. Thank you so much! 💯

Reply:
All thanks to @tony-griffin for his hard work on this one :)